frontend/src/hooks/useUser.ts

```diff
@@ -34,5 +33,26 @@ export default function useUser() {
         })
     }
 
-    return {registerUser, loggedInUser, login}
+    function logout(): Promise<void> {
+        return getToken().then(() => {
+            logoutUser()
+                .then(() => setLoggedInUser(initialUser))
+                .catch(console.error)
+        })
+    }
+
+    function edit(userRequest: UserRequest): Promise<void> {
+        return getToken().then(() => {
+            updateUser(userRequest)
+                .then(data => setLoggedInUser(data))
+        })
+    }
+
+    function deletee(id: string): Promise<void> {
```

Comment:
Ups, ein E zu viel

Reply:
kein e zu viel. delete ist ein reserviertes Keyword in typescript